sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -10,8 +10,8 @@
     requires transitive com.fasterxml.jackson.core;
     requires transitive com.fasterxml.jackson.databind;
 
-    requires com.fasterxml.jackson.dataformat.xml;
-    requires com.fasterxml.jackson.datatype.jsr310;
+    requires transitive com.fasterxml.jackson.dataformat.xml;
+    requires transitive com.fasterxml.jackson.datatype.jsr310;
```

Comment:
I thought requires transitive was only needed when the module is exposed in public API?

Reply:
No, `requires transitive` allows client libraries to access it if they just add `requires com.azure.core` to their respective module-info.java. A few client libraries like Cosmos, Storage and Search are not explicitly adding `requires com.fasterxml.jackson.dataformat.xml` or `requires  com.fasterxml.jackson.datatype.jsr310` which can be avoided with this change.